app/models/mail_delivery_failure.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: mail_delivery_failures
+#
+#  id                   :bigint           not null, primary key
+#  email_address        :string
+#  mailer_action_method :string
+#  mailer_class         :string
```

Comment:
It looks like these don't match the table, the comments are generated automatically when `db:migrate` is run but they should refer to these fields being `not null`.